--- a/src/sim/simulator.py
+++ b/src/sim/simulator.py
@@ -105,11 +105,11 @@ class Simulator:
     def _publish_satellite_ground_position(self, utcg_time, sim_time):
         lon, lat, alt = self.get_orbital_location(utcg_time)
         dispatcher.send(
             signal=TOPIC_SATELLITE_GROUND_POSITION,
             sender=str(self),
             data={'lon': lon, 'lat': lat, 'alt': alt},
-            time=datetime.datetime.fromtimestamp(utcg_time).isoformat(),
+            time=datetime.datetime.fromtimestamp(utcg_time, tz=datetime.timezone.utc).isoformat().replace("+00:00", "Z"),
             time_epsec=sim_time,
         )
